Azure Device Twin and RBAC docs (#1613)
mendersoftware/mender-docs: 08.Server-integration/04.Azure-IoT-Hub/docs.md

    category: docs
---

> Mender supports preparing devices to use Azure IoT Hub. Assuming you already have Mender in place on the device, it is automatically created in Azure IoT Hub and applications running on the device get access to the Azure IoT connection string. With Mender in place, you can simply start writing your Azure IoT application by reading the connection string from a well-defined place in the device. Secondly, the Device Twin from the Azure IoT Hub becomes available to manage from Mender.

Reviewer: ```suggestion
Mender supports preparing devices to use Azure IoT Hub. After Mender is installed on the device, it is automatically created in Azure IoT Hub, and applications running on the device get access to the Azure IoT connection string. With Mender in place, you can simply start writing your Azure IoT application by reading the connection string from a well-defined place on the device. Secondly, the Device Twin from the Azure IoT Hub becomes available to manage from Mender.
```
